contracts/ROSCA.sol

```diff
@@ -130,10 +150,47 @@ contract ROSCA {
 
       currentRound++;
       LogStartOfRound(currentRound);
+    } else if (currentRound == membersAddresses.length) {
```

Comment:
don't want to take chances in case of attacker managed to make currentRound > membersAddresses.length